tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -16,8 +16,9 @@
 		const branch = jQuery( '.head-ref' ).text();
 		const branchIsForked = branch.includes( ':' );
 		const branchIsMerged = $( '.gh-header-meta .State' ).text().trim() === 'Merged';
-		const query = 'jetpack-beta&branch=' + branch + '&shortlived&wp-debug-log&gutenberg';
+		const isGutenbergPr = $( '.discussion-sidebar .sidebar-labels .labels' ).children( 'a' ).attr( 'title' ) === 'Gutenberg';
```

Comment:
Right now the following code don't work with PR's where `Gutenberg` label isn't first in the list for example in #10347. You search for specific label right away. 

```suggestion
		const isGutenbergPr = $( '.discussion-sidebar .sidebar-labels .labels a[title='Gutenberg']' ).length;
```